x-pack/plugins/fleet/server/services/package_policy.ts

```diff
@@ -1382,7 +1384,9 @@ export async function incrementPackageName(
     ? packagePolicyData.items
         .filter((ds) => Boolean(ds.name.match(pkgPoliciesNamePattern)))
         .map((ds) => parseInt(ds.name.match(pkgPoliciesNamePattern)![1], 10))
-        .sort()
+        .sort((a: number, b: number) => {
+          return a - b;
```

Comment:
this fixes a bug, default sorting doesn't work correctly on numbers, e.g. `"1, 10, 9"` - this was giving an error when reaching `system-10` name in integrations, and the logic returned `system-10` again as incremented name